### PR TITLE
remove dragon and dinosaur

### PR DESCRIPTION
**Pull request by glitchcrab**
towards https://github.com/giantswarm/giantswarm/issues/17621

### CODE REVIEW

**Review by tfussell — APPROVED** [review]

**Comment by QuentinBisson** [comment]
Can you remove Davis as well?

File: events/events.go [hunk 1]
```diff
@@ -57,8 +57,6 @@ var environmentProjects = map[Environment][]string{
 	"beaver":    kvmProjectList,
 	"buffalo":   kvmProjectList,
 	"davis":     azureProjectList,
```
Comment (QuentinBisson) [inline events/events.go:59]:
```suggestion
```

**Comment by glitchcrab** [comment]
@QuentinBisson dropped Archon too as it is also dead

**Review by QuentinBisson — APPROVED** [review]

**Comment by QuentinBisson** [comment]
Thank you :D

**Pull request merged**